src/dashboards/teachers/common.libsonnet

```diff
@@ -70,18 +90,40 @@ local common = import '../common.libsonnet';
       hide='variable',
       refresh='time'
     ),
-    event_group_interval: template.custom(
-      name='EVENT_GROUP_INTERVAL',
-      current='30',
-      label='Event group interval',
-      query='1,10,20,30,60,120,180,300,600',
+    title: template.new(
+      name='TITLE',
+      current='all',
+      label='Title',
+      datasource=common.datasources.edx_app,
+      query=$.queries.course_title,
+      hide='variable',
       refresh='time'
     ),
-    statements_interval: template.custom(
-      name='STATEMENTS_INTERVAL',
-      current='7d',
-      label='Statements interval',
-      query='1d,7d,14d,21d,28d',
+    start_date: template.new(
+      name='START_DATE',
+      current='all',
+      label='Start Date',
+      datasource=common.datasources.edx_app,
+      query=$.queries.course_start_date,
+      hide='variable',
+      refresh='time'
+    ),
+    end_date: template.new(
+      name='END_DATE',
+      current='all',
+      label='End Date',
+      datasource=common.datasources.edx_app,
+      query=$.queries.course_end_date,
+      hide='variable',
+      refresh='time'
+    ),
```

Comment:
Maybe we can set the dashboard default analysis period to those values (start/end dates)?